src/Clients/StripeClient.php

```diff
@@ -52,4 +53,22 @@ public function getBalance()
 
         }
     }
+
+    public function refundCharge($chargeId)
+    {
+        try {
+            return Refund::create(['charge' => $chargeId], ['api_key' => $this->apiKey]);
+        } catch (Exception $e) {
+
```

Comment:
Does anyone know why we're swallowing these errors? Feels like it would be better to let them throw, and then make sure we handle the errors by returning a nice JSON error message?

Reply:
@mattstauffer  - What do you think about me handling these Exceptions (and the Vue error messages you mention above) in a different PR? I'm planning on moving testing to a fake gateway, rather than using the API directly, in order to speed up tests and this feels like it might make more sense there.

Reply:
@mattstauffer - How would you feel about me doing this in a new PR? I'm thinking about moving tests to a fake gateway, to speed up tests, and this feels like it _might_ make more sense there.

Reply:
Absolutely fine! :)